app.js

```diff
@@ -54,13 +60,24 @@ app.use('/account', account);
 app.use('/play', play);
 app.use('/api', api);
 app.use('/search', search);
-
+app.set('trust proxy', '192.168.12.77'); // specify a single subnet
 // configure error handlers
 require('./config/errorHandlers.js')(app);
 
+
+
+var url_path='192.168.12.77';
```

Comment:
Host and port configuration is not related to adding time taken by each player.
Please send this change in a separate pull request.
I have created an issue for that. see here: #7